packages/p2p/src/pages/my-profile/payment-methods/edit-payment-method-form/edit-payment-method-form.tsx

```diff
@@ -59,8 +60,12 @@ const EditPaymentMethodForm = () => {
         } else if (mutation_status === 'error') {
             my_profile_store.setAddPaymentMethodErrorMessage(mutation_error.message);
             showModal({ key: 'AddPaymentMethodErrorModal', props: {} });
+            if (setSubmittingRef.current) {
```

Comment:
can we pass `setSubmitting` to `setAddPaymentMethodErrorMessage` instead of creating a new ref?